node/src/command.rs

```diff
@@ -309,21 +310,45 @@ pub fn run() -> Result<()> {
 
 			Ok(())
 		},
-		Some(Subcommand::Benchmark(cmd)) =>
-			if cfg!(feature = "runtime-benchmarks") {
-				let runner = cli.create_runner(cmd)?;
-				let chain_spec = &runner.config().chain_spec;
-				with_runtime_or_err!(chain_spec, {
-					{
-						runner.sync_run(|config| cmd.run::<Block, Executor>(config))
+		Some(Subcommand::Benchmark(cmd)) => {
+			let runner = cli.create_runner(cmd)?;
+			let chain_spec = &runner.config().chain_spec;
+			// Switch on the concrete benchmark sub-command-
+			with_runtime_or_err!(chain_spec, {
+				{
+					match cmd {
+						BenchmarkCmd::Pallet(cmd) =>
```

Comment:
This is realllllyyyy important to note as it changes the benchmark command. You now need to add `pallet` as a subcommand:

./target/release/oak-collator benchmark pallet \
--chain dev \
--execution wasm \
--wasm-execution compiled \
--pallet pallet_valve \
--extrinsic "*" \
--repeat 64 \
--steps 20 \
--output ./pallets/valve/src/raw-weights.rs

Reply:
might be good to run benchmarks so the command is saved in the weights file? or just add it manually, or in a wiki or something